Sources/PaymentsCore/Networking/Models/Responses/AccessTokenResponse.swift

```diff
@@ -2,7 +2,7 @@
 
 public struct AccessTokenResponse: Codable {
     let scope: String
-    let accessToken: String
+    public let accessToken: String
```

Comment:
q: can we remove public on this property?

Reply:
At a second glance we could move this entire model into the Demo application. That may solve access issues.